modules/administration/pages/master-fingerprint.adoc

```diff
@@ -14,19 +24,23 @@ salt-key -F master
 ----
 +
 
-On your client, open the [path]``/etc/salt/minion`` configuration file.
+On your client, open the [path]``/etc/salt/minion`` or [path]``/etc/venv-salt-minion/minion.d/custom.conf`` configuration file.
 Uncomment the following line and enter the master's fingerprint replacing the example fingerprint:
 +
 
 ----
 master_finger: 'ba:30:65:2a:d6:9e:20:4f:d8:b2:f3:a7:d4:65:11:13'
 ----
 
-. Restart the salt-minion service:
+. Restart the salt-minion or venv-salt-minion service.
+  For venv-salt-minion, run:
 +
 
 ----
-# systemctl restart salt-minion
+systemctl restart vent-salt-minion
 ----
 
+In case of using {salt} Bundle, instead of [literal]``salt-minion`` use [literal]``venv-salt-minion``.
```

Comment:
This statement should be the opposite, isn't it? As we already provided the example for `venv-salt-minion`:

Or maybe we should keep this statement here, but then provide the above example of restarting the service with the classic Salt example: `systemctl restart salt-minion`